tests/testthat/test-extractVariableGroups.R

```diff
@@ -27,8 +27,12 @@ test_that("check that extractVariableGroups correctly extracts variable groups",
   res2 <- res1
   names(res2) <- sub("a 2|a 3","a",names(res1))
   names(res2) <- sub("a2 4","a2 (unit)",names(res2))
+  res3 <- list(`a|1` = c("a|1|aa", "a|1|bb", "a|1|cc"),
+               `a|3` = c("a|3|aa","a|3|bb"),
+               a = c("a|1", "a|3", "a|bla", "a|blu", "a|ble", "a|blet"))
```

Comment:
I believe a2 should show up here as well in `res3` with something like `a2 = c("a2|wunit (unit)" ,   "a2|wunit2 (unit 2)"`

Reply:
No, because `a2` doesn't exist, only `a2 (unit)`, the tool cannot handle variable names with units. We use it only with data.frames. Should I add a warning?

Reply:
yes, please add a warning...especially as the unit-case works properly in the example above (with +)